packages/ui-extensions-server-kit/.gitignore

```diff
@@ -1,3 +1,5 @@
 *.d.ts
 !typings.d.ts
+!index.d.ts
```

Comment:
Can you clarify why we need to check these files into git? Can we not run tsc to emit only the types before running Vite?

Reply:
I'll look into generating them during build. They are just entry points to the built code but I agree it's better to add them in build time.